Utilities/DB.php

```diff
@@ -12,7 +13,15 @@ class DB
      */
     public function dbConnect()
     {
-        $dbconnect = new PDO($this->serverdb, $this->username, $this->password);
+        $dbconnect = new PDO($this->server .$this->db, $this->username, $this->password);
```

Comment:
Non-blocking: missing a space after concatenation dot

Reply:
on it